src/mn_setup.c

```diff
@@ -2100,7 +2100,9 @@ static const char **M_GetMouseAccelStrings(void)
     static const char *strings[MOUSE_ACCEL_STRINGS_SIZE];
     char buf[8];
 
-    for (int i = 0; i < MOUSE_ACCEL_STRINGS_SIZE; ++i)
+    strings[0] = M_StringDuplicate("Off");
```

Comment:
It's not necessary to duplicate static strings in C, just `strings[0] = "Off"` will work. From what I understand, the duplication happens under the hood anyway. I guess it's a matter of style.

Reply:
Oops, sorry. Feel free to update that if you'd like. I'm away from my computer for now.

Reply:
> From what I understand, the duplication happens under the hood anyway.

I think string literals are always `static const` in C, so the pointer never gets lost.

Reply:
> I think string literals are always `static const` in C, so the pointer never gets lost.

It seems you are right: https://stackoverflow.com/questions/2589949/string-literals-where-do-they-go